src/components/DoesMeet.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import styled from "styled-components";
+import CheckSecond from "./icons/CheckSecond";
+
+const DoesMeet = styled(CheckSecond)`
+  fill: #6be895;
+`;
+
+export default function MeetIcon() {
```

Comment:
Im not sure but MeetIcon is a little bit confusing. Do you mean match ? or meet as in "2 persons meet somewhere" ? I'd rather use MeetupIcon if u meant that.